src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -666,7 +667,8 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
             Func<TypeSymbol, T, bool, bool>? typePredicate,
             T arg,
             bool canDigThroughNullable = false,
-            bool useDefaultType = false)
+            bool useDefaultType = false,
+            bool visitCustomModifiers = false)
```

Comment:
>visitCustomModifiers [](http://example.com/codeflow?start=17&length=20)

I think that ```visitCustomModifiers``` and ```canDigThroughNullable``` are incompatible. I.e. they should not be both true at the same time. Let's add an assert about that.

Reply:
Sure. I updated the implementation following the feedbacks provided in this method, please check the method again.